modular_skyrat/modules/reagent_forging/code/forge.dm

```diff
@@ -557,6 +560,16 @@
 
 /// Handles weapon reagent imbuing
 /obj/structure/reagent_forge/proc/handle_weapon_imbue(obj/attacking_item, mob/living/user)
+	//This code will refuse all non-ashwalkers & non-icecats from imbuing
+	if(!ishuman(user))
+		to_chat(user, span_danger("It is impossible for you to imbue!")) //maybe remove (ashwalkers & icecats only) after some time
```

Comment:
the comments here and below don't mean anything

Reply:
it says exactly what it does-- prevents non-ashwalkers and non-icecats from imbuing; yes, there is a space between the human check and the species check, Golden asked for that.